.storybook/preview.tsx

```diff
@@ -18,6 +18,7 @@ const AppDecorator: Exclude<Preview["decorators"], undefined>[0] = (
       <ReactAriaI18nProvider locale={getLocale(l10nBundles)}>
         <div
           className={`${inter.className} ${inter.variable} ${metropolis.variable}`}
+          style={{ height: "100%" }}
```

Comment:
I was able to remove this wrapper in https://github.com/mozilla/blurts-server/pull/3190, but until that's merged, this `style` attribute is needed as well.

Reply:
Hahaha, I had forgotten about this and was confused about the conflict I ran into, but this comment saved me :sweat_smile: